captum/attr/_utils/common.py

```diff
@@ -166,7 +171,22 @@ def _zeros(inputs):
     shape as the `inputs`
 
     """
-    return tuple(0 * input for input in inputs)
+    return tuple(0.0 for input in inputs)
+
+
+def _tensorize_baseline(inputs, baselines):
+    assert isinstance(inputs, tuple) and isinstance(baselines, tuple), (
```

Comment:
Can we potentially combine this with _expand_baselines_based_on_inputs to perform all baseline expansions in one call?